spec/models/email_schedule_spec.rb

```diff
@@ -6,6 +6,25 @@
   it { is_expected.to validate_inclusion_of(:mailer_name).in_array(EmailSchedule::MAILERS.keys.map(&:to_s)) }
   it { is_expected.to validate_presence_of(:scheduled_at) }
 
+  describe "custom validations" do
+    describe "#validate_future_schedule_date" do
+      let(:past_scheduled_email) { build(:seed_email_factory, scheduled_at: Date.yesterday) }
+      let(:future_scheduled_email) { build(:seed_email_factory, scheduled_at: Date.tomorrow) }
+
+      it "adds an error when scheduled_at is not in the future" do
+        past_scheduled_email.valid?
+
+        expect(past_scheduled_email.errors[:scheduled_at]).to include("The schedule date must be in the future")
+      end
+
```

Comment:
could we add a spec to show today? might need a travel_to to help with that?  We have to allow changes to happen on the day prior to the midday send so would be good to demonstrate it

Reply:
@tonyheadford the schedule date needs to be in the future according to the ACs.

We could change it to allow schedules on the date prior to the midday, but same day schedules prior to midday seem like edge cases to me. I would keep it simple for now and change it in the future if need to.